### PR TITLE
Simple changes and organizing

### PR DESCRIPTION
**Pull request by Ilithor**

### CODE REVIEW

File: src/pages/login.js [hunk 1]
```diff
@@ -65,13 +65,8 @@ const LoginPage = ({ classes = {}, UI = {}, history, loginUserAction }) => {
...
 
-const mapStateToProps = state => {
-  const user = state.user;
-  const UI = state.UI;
-  return {
-    user,
-    UI,
-  };
+const mapStateToProps = ({ UI }) => {
+  return { UI };
```
Comment (jeffersoneagley) [inline src/pages/login.js]: can be simplified to 
```
=> ({ UI })

File: src/pages/register.js [hunk 2]
```diff
@@ -68,13 +68,8 @@ const RegisterPage = ({
...
 
-const mapStateToProps = state => {
-  const user = state.user;
-  const UI = state.UI;
-  return {
-    user,
-    UI,
-  };
+const mapStateToProps = ({ UI }) => {
+  return { UI };
```
Comment (jeffersoneagley) [inline src/pages/register.js]: same as above

File: src/pages/login.js (hunk 1 above)
Comment (Ilithor) [reply src/pages/login.js]: Fixed. Also, found various other components that were also simplified with the same process.

File: src/pages/register.js (hunk 2 above)
Comment (Ilithor) [reply src/pages/register.js]: fixed

**Pull request merged**